webapp/Gemfile

```diff
@@ -50,7 +50,7 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'devise'
-gem 'bcrypt', '~> 3.1.11'
+gem 'bcrypt', '~> 3.1.11', 'platforms::ruby'
```

Comment:
This should not be quoted, it should only be `, platforms: :ruby`. Did you not run `bundle install` after making this change to test? I'm certain it will fail.